### PR TITLE
Update ome-faq.yml

### PR DESCRIPTION
**Pull request by samschan-ms**
updated that OME branding needs a mail sent from inside the cloud organization. @KCCross to review

### CODE REVIEW

File: microsoft-365/compliance/ome-faq.yml [hunk 1]
```diff
@@ -190,7 +190,7 @@ sections:
       - question: |
           Can I customize encrypted messages with my company branding?
         answer: |
-          Yes! For information on customizing email messages and the OME portal, see Add your organization's brand to your encrypted messages. See [Add your organization's brand to your encrypted messages](add-your-organization-brand-to-encrypted-messages.md).
+          Yes, for mail sent from an Exchange Online mailbox in your organization! For information on customizing email messages and the OME portal, see Add your organization's brand to your encrypted messages. See [Add your organization's brand to your encrypted messages](add-your-organization-brand-to-encrypted-messages.md).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ome-faq.yml]:
```suggestion
          Yes, for mail sent from an Exchange Online mailbox in your organization! For information on customizing email messages and the OME portal, see [Add your organization's brand to your encrypted messages](add-your-organization-brand-to-encrypted-messages.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @samschan-ms, kindly commit the suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@KCCross Gentle reminder

**Pull request merged**